os-eap7-openshift/added/standalone.conf

```diff
@@ -12,3 +12,8 @@ fi
 
 # Append to JAVA_OPTS. Necessary to prevent some values being omitted if JAVA_OPTS is defined directly
 JAVA_OPTS="$JAVA_OPTS $JAVA_OPTS_APPEND"
+
+source $JBOSS_HOME/bin/launch/java-alternatives.sh
+
+export JAVA_HOME="$(set_java_alternatives)"
+
```

Comment:
This appears to be EAP only but I don't see the yaml change to pull in the java-alternatives module, is there another PR associated with this?

Reply:
I have just done EAP7 initially to get it reviewed. The java-alternatives module is already in the jboss-eap-7-openshift-image image.yaml, but I will move to os-java-run and update the corresponding files for all images (e.g. standalone.conf). I was really hoping to find a way where I didn't have to touch files specific to each image but that seems like the only way.

Reply:
This is the PR that installs the 32bit JVM: https://github.com/jboss-container-images/jboss-openjdk-image/pull/7

Reply:
+1 I found that yesterday when reviewing this PR